doc/api/prep_data/feature_store.rst

```diff
@@ -72,3 +72,11 @@ Inputs
 .. autoclass:: sagemaker.feature_store.inputs.FeatureValue
     :members:
     :show-inheritance:
+
+
+Dataset Builder
+******
```

Comment:
Title underline should match characters of title, please correct

Reply:
Thanks, done!